src/zope/app/server/tests/test_ftp.py

```diff
@@ -15,12 +15,17 @@
 """
 import unittest
 
+try:
+    from cStringIO import StringIO
+except ImportError:
+    from io import StringIO
```

Comment:
Because this is being used as the input stream for an FTPRequest, I suspect this should be `io.BytesIO` on all platforms. The string literal down below will need a `b` prefix.